content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -77,46 +94,68 @@ Once the credit note has been posted, a :guilabel:`Credit Notes` smart button wi
 .. seealso::
    :doc:`/applications/finance/accounting/customer_invoices/credit_notes`
 
+.. _helpdesk/advanced/after_sales/coupons:
+
 Generate coupons from a ticket
 ==============================
 
-Coupons can be used to alter the price of products or orders. The usage constraints of a coupon are
-defined by conditional rules. *Coupon Programs* are configured in the *Sales* or *Website*
-applications.
+Coupons can be used to alter the price of products or orders. Conditional rules define the usage
+constraints of a coupon. *Coupon Programs* are configured in the *Sales*, *Point of Sale*, or
+*Website* applications.
 
-.. note::
-   The *eCommerce* module must be installed in order to create coupon codes from the *Website*.
+.. important::
+   The *eCommerce* module must be installed to create coupon codes from the *Website*.
 
 To generate a coupon, open a *Helpdesk* ticket and click on the :guilabel:`Coupon` button in the
-upper left corner. Select an option from the :guilabel:`Coupon Program` drop-down menu, then click
-:guilabel:`Generate`.
+top-left corner. Select an option from the :guilabel:`Coupon Program` drop-down menu.
 
 .. image:: after_sales/after-sales-generate-coupon.png
    :align: center
    :alt: View of a coupon generation window.
 
-The :guilabel:`Coupon Code` can be copied directly from the pop-up window (by clicking the
-:guilabel:`Copy` button), or sent in an email by clicking :guilabel:`Send`.
+.. note::
+   To create a new :guilabel:`Coupon Program`, navigate to :menuselection:`Sales app --> Products
+   --> Discount & Loyalty` and click :guilabel:`New`. To make the program available to share with
```

Comment:
In one of my runbots, I didn't see the "Discount & Loyalty' option in the drop-down. I am not sure what needs to be set to make that visible- but it could be mentioned if you want.

Reply:
I checked four runbots and two demo dbs and it was there on all of them, so I do not know what caused that issue, but I am going to assume it was a problem with that particular runbot and not the menu itself.